### DISCUSSION TITLE
conference.html example does not work

### DISCUSSION THREAD

**Discussion by cppmyjob** [ant-media/Ant-Media-Server#5303] [General]
The current conference.html example does not work on version 2.6 - an error occurs - no_active_streams_in_room. Also, when opening this example, it says "This conference solution is deprecated. Please use performance efficient Multitrack Conferencing". Should we now always use Multitrack Conferencing to organize a conference or is there a way to make conference.html work on the latest version of the server?

**Comment by Mohit-3196** [comment]
Hi @cppmyjob ,
How are you?

Yes, the conference.html sample page is deprecated and we have moved to the multitrack conference which is multitrack-conference.html, and in the near future, conference.html will be fully deprecated. 
The conference room with "legacy" mode is a normal conference room without multitrack. You can take a look at the conference.html sample page for the legacy mode.

Here is the websocket message for the conference.html
`sent message:{"command":"joinRoom","room":"room1","mode":"legacy"}`

Thank you

**Comment by Mohit-3196** [comment]
Hi @cppmyjob ,
Hope you are doing well.

I was wondering if my answer could help with your question.
Let me know if you were able to check multitrack-conference.html
If you have any issues/questions or if I can do anything for you please let me know.

Thank you
